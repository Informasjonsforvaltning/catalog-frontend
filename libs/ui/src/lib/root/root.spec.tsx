import { render } from '@testing-library/react';
import React from 'react';

import Root from './';

describe('Root', () => {
  
  it('should render successfully', () => {
    const { baseElement } = render(<Root />);
    expect(baseElement).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
  
});
