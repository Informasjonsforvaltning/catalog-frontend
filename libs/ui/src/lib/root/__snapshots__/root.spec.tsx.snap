// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root should render successfully 1`] = `
<body>
  <div>
    <div
      class="root"
    />
  </div>
</body>
`;
